packages/api/endpoints/collections.js

```diff
@@ -238,44 +223,26 @@ async function del(req, res) {
   const {
     collectionPgModel = new CollectionPgModel(),
     knex = await getKnexClient(),
-    esClient = await getEsClient(),
     collectionConfigStore = new CollectionConfigStore(
       process.env.system_bucket,
       process.env.stackName
     ),
   } = req.testContext || {};
 
   const { name, version } = req.params;
-  const collectionId = constructCollectionId(name, version);
-  const esCollectionsClient = new Search(
-    {},
-    'collection',
-    process.env.ES_INDEX
-  );
 
   try {
     await collectionPgModel.get(knex, { name, version });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
-      if (!(await esCollectionsClient.exists(collectionId))) {
-        log.info('Collection does not exist in Elasticsearch and PostgreSQL');
-        return res.boom.notFound('No record found');
-      }
-      log.info('Collection does not exist in PostgreSQL, it only exists in Elasticsearch. Proceeding with deletion');
-    } else {
-      throw error;
+      log.info(`Collection does not exist in PostgreSQL. Failed to delete collection with name ${name} and version ${version}`);
+      return res.boom.notFound('No record found');
```

Comment:
I think we still need the `else throw error`, right? If the error is not RecordDoesNotExist, I believe this will swallow the error unless there's a catch further up I'm not looking at (which is possible).

Reply:
All good. We can mark things as WIP and/or put them in Draft status in the future but not a big deal.